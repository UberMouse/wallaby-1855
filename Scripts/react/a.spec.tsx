const Enzyme = require('enzyme');
const Adapter = require('enzyme-adapter-react-16');
import { createSerializer } from 'enzyme-to-json';

expect.addSnapshotSerializer(createSerializer({
    map: json => ({
        ...json,
        props: json.props
    })
}));

Enzyme.configure({
    adapter: new Adapter()
});

import React from 'react';
import { mount } from 'enzyme';

const Test: React.StatelessComponent<{}> = () => <p>Test</p>;

it('works', () => {
    expect(mount(<Test />)).toMatchSnapshot();
});