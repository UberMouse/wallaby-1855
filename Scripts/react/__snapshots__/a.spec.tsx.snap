// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
<Test>
  <p>
    Test
  </p>
</Test>
------------HTML PREVIEW---------------
<Test>
  <p>
    Test
  </p>
</Test>
`;
